clients/src/main/java/org/apache/kafka/common/requests/RequestUtils.java

```diff
@@ -26,6 +26,9 @@
 import org.apache.kafka.common.resource.ResourceType;
 
 class RequestUtils {
```

Comment:
Is there a reason why this one is not final?

Reply:
@ijuma there was no particular reason, added `final` keyword here to